sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -59,9 +59,9 @@ export interface EventData {
 // @public
 export interface EventDataBatch {
     readonly count: number;
-    readonly maxSizeInBytes: number;
     // @internal
-    readonly _message: Buffer | undefined;
+    _generateMessage(): Buffer;
```

Comment:
This looks like a scarier change than it is. Note that it is marked internal and does not appear in the type definitions (neither did `_message`). This was only ever used internally.

Reply:
/cc @bterlson , I want to make sure you're ok with this change since it shows up in the api review file.